php-nginx/README.md

```diff
@@ -281,6 +281,13 @@ Note: this doesn't reflect any additional web server configuration.
   provide an easier way to install extensions.
 - PHP 7 installation is still missing various extensions including
   suhosin.
+- As a security measure, the `/app` directory is changed to being read-only
+  during the build. If you need to write to specific folders in your `/app` 
+  directory, you need to make it writeable to the `www-data` user as a separate
+  run-step in your Dockerfile.
```

Comment:
Hm. I totally see your arguments. - I would prefer it as well. But I also see what others do :)

IMHO the use case is more like the one I mentioned in #27: People use a framework, switch the document root anyways and just need the app to run the way it is intended to run. To accomplish this, the framework needs to be able to write into 1 or 2 locations. If it cannot, that's a bummer.

Besides that: a writable directory is a writable directory - wether it's in `/app` or not. - The key is the document root switch. Don't you think so?